src/schema/types/person.graphql

```diff
@@ -0,0 +1,21 @@
+type Person {
+  id: ID!
+  firstName: String!
+  lastName: String!
+  address: Address!
+}
+
+input PersonalInformationInput {
+ firstName: String!
+ lastName: String!
+ address: AddressInput!
+ dni: Int
+}
+
+extend type Query {
+  getPersonalInformation(userId: ID!): Person!
+}
+
+extend type Mutation {
+  setPersonalInformation(userId: ID!, personalInformation: PersonalInformationInput!): Person!
+}
```

Comment:
These queries are not how this Entity should be managed. This entity is always a part of a user, it doesn't exist on its own.
- The `getPersonalInformation` query should be a `person: Person` field in the `User` type. Note that I intentionally left this field as nullable. I don't remember if we're going to allow a user without a person or not. If all users should have a person information, then this field should be not null: `person: Person!`.
- The update of this information is part of an update of a `User`: `updateUser(input: UpdateUserInput!): User!`. Then, inside of `UpdateUserInput` you can have a field `person: UpdatePersonInput`.
- Same as the previous item for `Person` creation related to a `User`.
- The `Person` type should have a `user: User!` field if a `Person` always belongs to a user.